### PR TITLE
Training toggle on country level

### PR DESCRIPTION
**Pull request by reinhardt**
The three levels of switches (global, country, tool) are potentially confusing and at least the wording should be improved, but we don't have time for that right now.

Refs syslabcom/scrum#295

### CODE REVIEW

File: src/euphorie/client/browser/webhelpers.py [hunk 1]
```diff
@@ -165,9 +165,11 @@ def use_involve_phase(self):
     @property
     @memoize
     def use_training_module(self):
-        return api.portal.get_registry_record(
+        globally_enabled = api.portal.get_registry_record(
             "euphorie.use_training_module", default=False
         )
+        country_enabled = self.content_country_obj.enable_web_training
```
Comment (ale-rt) [inline src/euphorie/client/browser/webhelpers.py:171]: It is useless to get the `country_enabled` if globally_enabled is False.

File: src/euphorie/content/country.py [hunk 2]
```diff
@@ -99,6 +99,17 @@ class ICountry(model.Schema, IBasic):
...
+    enable_web_training = schema.Bool(
+        title=_("label_enable_web_training", default="Enable Web Based Training?"),
+        description=_(
+            "help_enable_web_training_country",
+            default="If this option is activated, an online training can be enabled on "
+            "OiRA tools in this country.",
+        ),
+        required=False,
+        default=True,
+    )
```
Comment (ale-rt) [inline src/euphorie/content/country.py:111]: It feels weird that the default for `euphorie.use_training_module` is `False` and this is `True`.

**Review by ale-rt — APPROVED** [review]
This is okish

File: src/euphorie/content/country.py (hunk 2 above)
Comment (reinhardt) [reply src/euphorie/content/country.py:111]: Yes, but they also do different things, which is worse, but no time to get it straight now.

File: src/euphorie/client/browser/webhelpers.py (hunk 1 above)
Comment (reinhardt) [reply src/euphorie/client/browser/webhelpers.py:171]: I'll change that, thx.

**Pull request merged**